src/views/ProjectView.vue

```diff
@@ -27,7 +24,13 @@ const authHeaders = {
 
 // toggle for optional data viewing, controlled by a v-switch
 let imageDisplayToggle = ref(true)
+let userFrames = ref(null)
 
+// Loads the user's Images from their profile into userImages ( currently just fetches all frames from archive regardless of proposal )
+const loadUserImages = async () => {
```

Comment:
Suggestion: We could put the reduction level as an argument to this function so that it's more reusable